ClientLibrary/lib/build.gradle

```diff
@@ -80,3 +87,4 @@ uploadArchives {
         }
     }
 }
+*/
```

Comment:
[Nit] Remove this.

Reply:
Yes, that shouldn't have reached the pull request